include/deal.II/base/utilities.h

```diff
@@ -270,6 +270,26 @@ namespace Utilities
   fixed_power (const T t);
 
   /**
+   * Calculate a fixed power (second, third, and forth order)  of a number.
+   *
+   * This functions have less code compared to
+   * <code>fixed_power@<dim@> (n)</code> and should be a bit faster to
+   * compile (less optimizations needed) and faster to run (in debug
+   * mode).
+   */
+  template <typename T>
+  T
+  pow2 (const T t);
```

Comment:
my personal opinion is that `fixed_power<2>()` is as readable as `pow2`, but I let others to decide on this issue.